numba/help/inspector.py

```diff
@@ -0,0 +1,388 @@
+"""
+This file contains `__main__` so that it can be run as a commandline tool.
+
+This file contains function to inspect Numba's support for a given Python
+module or a Python package.
+"""
+from __future__ import print_function
+
+import argparse
+import pkgutil
+import types as pytypes
+
+from numba._version import get_versions
+from numba.targets.registry import cpu_target
+from numba.tests.support import captured_stdout
+
+
+commit = get_versions()['full'].split('.')[0]
```

Comment:
This unfortunately seems to not work quite right in the docs subdir. From the `numba` top level dir:
```
$ pwd
<path>/numba

$ python -c 'from numba._version import get_versions; print(get_versions())'
{'version': '0.46.0dev0+21.g68c4634', 'full': '68c4634857cd1c97dca3f98ffb22487009546778'}
```
from the `numba/docs` subdir:
```
$ pwd
<path>/numba/docs

$ python -c 'from numba._version import get_versions; print(get_versions())'
{'version': '0.46.0.dev0+21.g68c4634-py3.7-linux-x86_64.egg', 'full': ''}
```
(note key `full` has no value) as a result, at least locally, I cannot get the links to render correctly with the `{commit}` present.

Reply:
how odd.  it works for me either way.  i think there'll need a default to fallback to the master branch.

Reply:
Fixed to fallback to 'master'